include/psa/crypto.h

```diff
@@ -361,15 +361,18 @@ void psa_reset_key_attributes(psa_key_attributes_t *attributes);
  * keys that can be opened with psa_open_key(). Such keys have a key identifier
  * in the vendor range, as documented in the description of #psa_key_id_t.
  *
- * The application must eventually close the handle with psa_close_key()
- * to release associated resources. If the application dies without calling
- * psa_close_key(), the implementation should perform the equivalent of a
- * call to psa_close_key().
+ * The application must eventually close the handle with psa_close_key() or
+ * psa_destroy_key() to release associated resources. If the application dies
+ * without calling one of these functions, the implementation should perform
+ * the equivalent of a call to psa_close_key().
  *
  * Some implementations permit an application to open the same key multiple
- * times. Applications that rely on this behavior will not be portable to
- * implementations that only permit a single key handle to be opened. See
- * also :ref:\`key-handles\`.
+ * times. If this is successful, each call to psa_open_key() will return a
```

Comment:
This change seems to obsolete the first sentence in this paragraph. The case of opening the same key multiple times has now been moved to a separate note below. Maybe the first sentence in this paragraph should also be moved there?

Reply:
There is an intended difference in the two paragraphs. The first paragraph is trying to state that:
1. An implementation is permitted to open a key more than once, but is not required to support this.
2. If it does, it must return distinct keys handles for each call.

The following paragraph is an informative note for application developers regarding the implication of depending on opening a key multiple times. It can be deduced from the rules above - as an implementation is permitted to refuse to open a key more than once - but the note just makes this effect clear from the app perspective, to reduce the risk of accidentally non-portable code.